### ISSUE TITLE
Columns not showing up

### ISSUE THREAD

**Issue by Yrlish** [isaac-mcfadyen/d1-jdbc-driver#1]
I added this driver to my WebStorm to be able to look inside my D1 database as I started to develop.

When I got to my second table, no columns was showing up, only one key representing the PK. This also means that all the database references for columns, etc, will be marked as invalid in the code.

I verified that my table was indeed created and correct by running the command `wrangler d1 execute DB --command "pragma table_info('table_name')"`.

The driver is outputting no messages, no errors.

**Comment by steebchen** [comment]
Yeah it doesn't show any columns except for the primary key (but the primary key is also lacking the type information)

<img width="180" alt="image" src="https://github.com/isaac-mcfadyen/d1-jdbc-driver/assets/5013932/c8f0967c-d980-4242-be5a-ed8a6cc5d520">
